gmail.js

```diff
@@ -1,40 +1,42 @@
 "use strict";
-const fs = require("fs");
-const readline = require("readline");
+const fs = require("fs").promises;
```

Comment:
TIL this exists. Cool stuff

Reply:
Yeah, super useful stuff when you want to adapt run "fs" commands async like you would an http request. The additional plus is it eliminates the need for callbacks (;